cruise.umple/src/GrammarParsing_Code.ump

```diff
@@ -36,41 +36,37 @@ class Analyzer
               for(Field mfield:clazz.getDeclaredFields())
               {
                 if(field.getName().equals(mfield.getName()))
-       	        {
-       	          try 
+                {
+                  try 
                   {
-	            classes.add(field.getType());
-	            variables.add(field.getName().substring(0,1).toUpperCase()+field.getName().substring(1));
+                    classes.add(field.getType());
+                    variables.add(field.getName().substring(0,1).toUpperCase()+field.getName().substring(1));
                     break;
-                  } 
-                  catch (IllegalArgumentException e)
-                  {
-	            e.printStackTrace();
-	          } 
-                  catch (NullPointerException nullPointerException)
-                  {
+                  }catch (IllegalArgumentException e){
+                    throw new IllegalStateException("Illegal argument encountered: " + e.getMessage());
+                  }catch (NullPointerException e){
+                    throw new IllegalStateException("Null state encountered: " + e.getMessage());
                   }
                 }
               }
               clazz = (Class<? extends Analyzer>)clazz.getSuperclass();
             }
           }
-          try 
-          {
+          try{
             List<Analyzer> originalChildren = analyzer.getChildren();
             analyzer = analyzer.getClass().newInstance();
             for(Analyzer child:originalChildren)
             {
               analyzer.addChild(child);
             }
-	    for(int i=0;i<classes.size();++i)
-	    {
-	      Object object = this.getClass().getMethod("get"+variables.get(i),new Class[]{}).invoke(this);
-	      analyzer.getClass().getMethod("set"+variables.get(i),new Class[]{classes.get(i)}).invoke(analyzer,object);
-	    }
-	  } catch (SecurityException | InstantiationException | IllegalAccessException | IllegalArgumentException | InvocationTargetException | NoSuchMethodException e) {
-	    e.printStackTrace();
-	  }
+            for(int i=0;i<classes.size();++i)
+            {
+              Object object = this.getClass().getMethod("get"+variables.get(i),new Class[]{}).invoke(this);
+              analyzer.getClass().getMethod("set"+variables.get(i),new Class[]{classes.get(i)}).invoke(analyzer,object);
+            }
+          } catch (SecurityException | InstantiationException | IllegalAccessException | IllegalArgumentException | InvocationTargetException | NoSuchMethodException e) {
+           throw new IllegalStateException("Error instantiating method: " + e.getMessage());
```

Comment:
Nothing wrong with being pedantic if it provides a better solution! I would prefer criticism if it contributes to my learning and/or helps to work toward something better :)

I wasn't even aware there was a propagation constructor for exceptions in Java since I've never had a reason to use them, so I'm happy to learn about that. I will update the PR in a few minutes with the changes.